bitnami/jupyterhub/values.yaml

```diff
@@ -0,0 +1,1592 @@
+        {{- end }}
+        {{- if .Values.singleuser.extraVolumeMounts }}
+        extraVolumeMounts: {{- include "common.tplvalues.render" ( dict "value" .Values.singleuser.extraVolumeMounts "context" $ ) | nindent 4 }}
+        {{- end }}
+        capacity: {{ .Values.singleuser.persistence.size }}
+        homeMountPath: /home/jovyan
+        dynamic:
+          storageClass: {{ include "jupyterhub.storageClass" . }}
+          pvcNameTemplate: {{ include "common.names.fullname" . }}-claim-{username}{servername}
+          volumeNameTemplate: {{ include "common.names.fullname" . }}-volume-{username}{servername}
+          storageAccessModes: {{- include "common.tplvalues.render" ( dict "value" .Values.singleuser.persistence.accessModes "context" $ ) | nindent 8 }}
+      image:
+        name: {{ include "jupyterhub.hubconfiguration.imageEntry" ( dict "imageRoot" .Values.singleuser.image "global" $) }}
+        tag: {{ .Values.singleuser.image.tag }}
+        pullPolicy: {{ .Values.singleuser.image.pullPolicy }}
+        pullSecrets: {{- include "jupyterhub.imagePullSecrets" . | nindent 8 }}
+      startTimeout: 300
+      {{- /* We need to replace the Kubernetes memory/cpu terminology (e.g. 10Gi, 10Mi) with one compatible with Python (10G, 10M) */}}
+      cpu:
+        limit: {{ regexReplaceAll "([A-Za-z])i" (default "" .Values.singleuser.resources.limits.cpu)  "${1}"}}
+        guarantee: {{ regexReplaceAll "([A-Za-z])i" (default "" .Values.singleuser.resources.requests.cpu) "${1}" }}
+      memory:
+        limit: {{ regexReplaceAll "([A-Za-z])i" (default "" .Values.singleuser.resources.limits.memory) "${1}" }}
+        guarantee: {{ regexReplaceAll "([A-Za-z])i" (default "" .Values.singleuser.resources.requests.memory) "${1}" }}
+      {{- if .Values.singleuser.command }}
+      cmd: {{- include "common.tplvalues.render" (dict "value" .Values.singleuser.command "context" $) | nindent 12 }}
+      {{- else }}
+      cmd: jupyterhub-singleuser
+      {{- end }}
+      defaultUrl:
+    cull:
+      enabled: true
+      users: false
+      removeNamedServers: false
+      timeout: 3600
+      every: 600
+      concurrency: 10
+      maxAge: 0
+
+  ## Hub Container port
+  ## @param hub.containerPort Hub container port
+  ##
+  containerPort: 8081
+
+  ## ConfigMap with hub init scripts
+  ## @param hub.existingConfigmap Configmap with Hub init scripts (replaces the scripts in templates/hub/configmap.yml)
+  ##
+  existingConfigmap:
+
+  ## Secret with hub configuration
+  ## @param hub.existingSecret Secret with hub configuration (replaces the hub.configuration value) and proxy token
+  ##
+  existingSecret:
+
+  ## Command and args for running the container (set to default if not set). Use array form
+  ## @param hub.command Override Hub default command
+  ## @param hub.args Override Hub default args
+  ##
+  command: []
+  args: []
+
+  pdb:
+    ## @param hub.pdb.enabled Deploy Hub PodDisruptionBudget
+    ##
+    enabled: false
```

Comment:
Use "create" for consistency with other charts

Reply:
Done